agent/proxycfg/connect_proxy.go

```diff
@@ -628,16 +628,17 @@ func (s *handlerConnectProxy) handleUpdate(ctx context.Context, u UpdateEvent, s
 	return nil
 }
 
-// hcpMetricsConfig represents the basic opaque config values for pushing telemetry to HCP.
-type hcpMetricsConfig struct {
-	// HCPMetricsBindSocketDir is a string that configures the directory for a
+// metricsCollectorConfig represents the basic opaque config values for pushing telemetry to
+// a consul telemetry collector.
+type metricsCollectorConfig struct {
+	// MetricsCollectorBindSocketDir is a string that configures the directory for a
 	// unix socket where Envoy will forward metrics. These metrics get pushed to
-	// the HCP Metrics collector to show service mesh metrics on HCP.
-	HCPMetricsBindSocketDir string `mapstructure:"envoy_hcp_metrics_bind_socket_dir"`
+	// the Consul Telemetry collector.
+	MetricsCollectorBindSocketDir string `mapstructure:"envoy_metrics_collector_bind_socket_dir"`
```

Comment:
@clly metrics felt maybe too generic, so I added "collector". Final name:`envoy_metrics_collector_bind_socket_dir`

Reply:
@freddygv we also discussed just calling it `envoy_telemetry_collector_bind_socket_dir` and re-using the socket for logs etc. We weren't sure if that would be a good idea as that could be a breaking change in the future or confusing if we can't send logs to the same socket. Let us know if you have any thoughts here!